kubernetes/istio_details_service.go

```diff
@@ -1511,13 +1511,12 @@ func PeerAuthnHasMTLSEnabled(peerAuthn IstioObject) (bool, string) {
 		if mtlsMap, ok := mtls.(map[string]interface{}); ok {
 			if modeItf, found := mtlsMap["mode"]; found {
 				if mode, ok := modeItf.(string); ok {
-					return true, mode
+					return mode == "STRICT" || mode == "PERMISSIVE", mode
```

Comment:
Side comment not blocker.
At some point we want to revisit these "business" methods in the kubernete package.
But no action for this PR.